### PR TITLE
Update: Consolidate mirage delivery rules route

### PR DESCRIPTION
**Pull request by Xuefeng-Zhu**

### CODE REVIEW

**Comment by cythrawll** [comment]
Can we wire these up with navi-app?

**Comment by Xuefeng-Zhu** [comment]
@cythrawll Does navi app support this feature?

**Comment by cythrawll** [comment]
It doesn't . it's more of a question if we want to add it.  It might need too much underlying infrastructure that we might not want to show this feature in navi-app.

File: packages/core/app/mirage/routes/delivery-rules.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import Mirage from 'ember-cli-mirage';
 
 const TIMESTAMP_FORMAT = 'YYYY-MM-DD HH:mm:ss';
 
-export default function() {
+export default function(deliveryType) {
```
Comment (jkusa) [inline packages/core/app/mirage/routes/delivery-rules.js]: Can we get get this from `deliveryRule.deliveryType`?

Comment (Xuefeng-Zhu) [reply packages/core/app/mirage/routes/delivery-rules.js]: updated

**Pull request merged**